src/daemon.rs

```diff
@@ -122,20 +122,19 @@ pub enum NotifyState {
 
 impl fmt::Display for NotifyState {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let msg = match *self {
-            NotifyState::Buserror(ref s) => format!("BUSERROR={}", s),
-            NotifyState::Errno(e) => format!("ERRNO={}", e),
-            NotifyState::Fdname(ref s) => format!("FDNAME={}", s),
-            NotifyState::Fdstore => "FDSTORE=1".to_string(),
-            NotifyState::Mainpid(ref p) => format!("MAINPID={}", p),
-            NotifyState::Other(ref s) => s.clone(),
-            NotifyState::Ready => "READY=1".to_string(),
-            NotifyState::Reloading => "RELOADING=1".to_string(),
-            NotifyState::Status(ref s) => format!("STATUS={}", s),
-            NotifyState::Stopping => "STOPPING=1".to_string(),
-            NotifyState::Watchdog => "WATCHDOG=1".to_string(),
-            NotifyState::WatchdogUsec(u) => format!("WATCHDOG_USEC={}", u),
-        };
-        write!(f, "{}", msg)
```

Comment:
Note to self: this change should avoid the overhead of doing two formatting steps (one of them unnecessary).